### DISCUSSION TITLE
Auth code does not exist or could not be found (transfer out)

### DISCUSSION THREAD

**Discussion by manzinello** [vercel/community#6835] [Help] [answered]
### Summary

I need to transfer a domain out, I try to use the "Transfer out" feature but in the dialog appears this text:

_Please wait while we fetch the auth code for [DOMAIN]...
An auth code does not exist or could not be found for [DOMAIN]. This TLD may not support auth codes. Please contact support._

The domain is a **.link**

What can I do?

### Steps to Reproduce

Simply clic on the Transfer out from the domain page

**Comment by amyegan** [comment]
Hey @manzinello. If you let me know the domain, I can look into this for you.

**Reply by manzinello** [reply, marked as answer]
I'm so sorry, I've made a mistake, the domain was already transferred, my fault. Sorry.

**Reply by amyegan** [reply]
No worries! 😄
